checkout/frames-ios
# Implement tokenisation for security code component

## Issue

JIRA-2139

## Proposed changes

Implement tokenisation capability for the security code component.

## Test Steps

> 1. Go to the main screen
> 2. Tap on `Security Code Component` button
> 3. Fill in the fields and tap `Pay Now` button. See the token is received.

## Checklist

_Put an `x` in the boxes that apply. You can also fill these out after creating the PR. If you're unsure about any of them, don't hesitate to ask. We're here to help! This is simply a reminder of what we are going to look for before merging your code._
